code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1079,7 +1079,7 @@
 	boiling_point = 20.271
 	color = "#808080" // rgb: 128, 128, 128
 	taste_mult = 0
-	pH = 0.1//Now I'm stuck in a trap of my own design. Maybe I should make -ve pHes? (not 0 so I don't get div/0 errors)
+	pH = 7//Now I'm stuck in a trap of my own design. Maybe I should make -ve pHes? (not 0 so I don't get div/0 errors)
```

Comment:
This is the only thing I'm hesitant on. Why are we changing this?
Isn't this correct?

Reply:
Ah there the github is, carl check dms I gave a lengthy explanation. Tiny nitpick. pH for hydrogen should be removed completely tbf as well as for several other substances that have pH values assigned to them. pH isnt really fully applicable. The chemistry system from Fermi means well. It just has some weird fucky things in it if you worked a bunch in that field.